onnxruntime/core/providers/cuda/cuda_execution_provider.h

```diff
@@ -20,6 +21,7 @@ namespace onnxruntime {
 // Logical device representation.
 class CUDAExecutionProvider : public IExecutionProvider {
  public:
+  static const int DEFAULT_REGULAR_RUNS_BEFORE_CUDA_GRAPH_CAPTURE = 1; // one regular run is needed before graph capture for the necessary memory allocations.
```

Comment:
Does this have to be static and a public member ? Can it just be like any other regular private member of the EP ?

Reply:
I thought it was the normal practice for this kind of default value. Just changed it to be a private member.